### ISSUE TITLE
`SLF001` should not emit error for `os._exit()`

### ISSUE THREAD

**Issue by dosisod** [astral-sh/ruff#6483]
The [`os._exit()`](https://docs.python.org/3/library/os.html#os._exit) function is often used to exit without doing any cleanup, either because you don't have any resources to cleanup, or you're running in another thread/subprocess like the docs describe.

Ruff shouldn't emit a `SLF001` error because it isn't "private" per-se: it's well documented, and the `_` prefix is probably indented to encourage people to use `sys.exit()` unless they need `os._exit()`.

Example:

```python
import os

os._exit(1)
```

Ruff output:

```
$ ruff --version
ruff 0.0.284

$ ruff x.py
x.py:3:1: SLF001 Private member accessed: `_exit`
Found 1 error.
```

Expected: no error.
